llvm/lib/CodeGen/ValueTypes.cpp

```diff
@@ -181,6 +181,7 @@ std::string EVT::getEVTString() const {
   case MVT::Metadata:  return "Metadata";
   case MVT::Untyped:   return "Untyped";
   case MVT::funcref:   return "funcref";
+  case MVT::exnref:    return "exnref";
```

Comment:
This causes a clang-format error, but this is in line with what's there, so won't fix it